protos/pbft_message.proto

```diff
@@ -56,9 +56,9 @@ message PbftMessage {
 // View change message, for when a node suspects the primary node is faulty
 message PbftViewChange {
   // Message information
-  PbftMessageInfo info = 1;
+  PbftMessageInfo info = 10;
```

Comment:
It was an attempt at having the protobuf parsing be easier, since protobuf will happily parse one message type with `PbftMessageInfo info = 1;` as another message type with the same field. Unfortunately since we're using `proto3`, this didn't end up making the parsing easier, since any missing fields will just get silently ignored, and proto3 dropped support for `required`. Long story short, it didn't end up improving parsing at all, and has been removed.